project_scrum/models/project.py

```diff
@@ -33,10 +33,14 @@ class ProjectSprint(models.Model):
     is_current_sprint = fields.Boolean('Is Current Sprint')
     is_previous_sprint = fields.Boolean('Is Previous Sprint')
 
-    @api.one
     def _task_count(self):
-        tasks = self.env['project.task'].search([('sprint_id', '=', self.id)])
-        self.task_count = len(tasks)
+        tasks = self.env['project.task'].read_group(
+            [('sprint_id', 'in', self.ids)],
+            ['sprint_id'],
+            ['sprint_id'])
```

Comment:
I'd love to see keywords here so the reader can easily see which is the domain, which is the grouping, etc.

Reply:
I added keywords